src/tmxrasterizer/tmxrasterizer.cpp

```diff
@@ -89,11 +89,35 @@ void TmxRasterizer::drawMapLayers(MapRenderer &renderer,
 
         const TileLayer *tileLayer = dynamic_cast<const TileLayer*>(layer);
         const ImageLayer *imageLayer = dynamic_cast<const ImageLayer*>(layer);
+        const ObjectGroup *objectGroup = static_cast<const ObjectGroup*>(layer);
```

Comment:
I can see that due to `shouldDrawLayer` excluding the group layer and there only being a limited number of layers, this potentially wrongly cast `ObjectGroup` is guaranteed to be only used when the cast is valid. Yet, I'd still prefer a `dynamic_cast` was used.

Or, since we're now handling almost every type of layer, maybe it's time to just do a `switch` on the layer type as done in the `MiniMapRenderer`.

I guess after this it would be worth moving the `MiniMapRenderer` into `libtiled`, so that it can also be used by the `tmxrasterizer` to avoid the code duplication.